### PR TITLE
Upgrade java9-beta to the just released to Java 1.9 Build 147

### PR DESCRIPTION
**Pull request by olivergierke**
*If there’s a checkbox you can’t complete for any reason, that's okay, just explain in detail why you weren’t able to do so.*

After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] The commit message includes the cask’s name and version.

### CODE REVIEW

**Comment by vitorgalvao** [comment]
Please fill the template.

File: Casks/java9-beta.rb [hunk 1]
```diff
@@ -1,5 +1,5 @@
 cask 'java9-beta' do
-  version '1.9,142'
+  version '1.9,147'
   sha256 '38bb89657ee6e9b33b4b488382de5a1379033a7bfa2c3a0bf71d8151f29023d4'
```
Comment (vitorgalvao) [inline Casks/java9-beta.rb]: `sha256` also needs to be set. You can find what it is by running `shasum -a 256` on the downloaded file.

Comment (olivergierke) [inline Casks/java9-beta.rb]: Sorry for the maybe stupid question. What do I replace {{cask_file}} with for a pending PR? I tried using `java9-beta` but that seems to be stupid obviously.

Comment (vitorgalvao) [inline Casks/java9-beta.rb]: That’s exactly what you replace it with. The path to your modified cask.

**Comment by olivergierke** [comment]
Updated the PR's title and added the new sha. Audit passed.

**Pull request merged**